src/components/FiatOnrampModal/index.tsx

```diff
@@ -125,7 +135,7 @@ export default function FiatOnrampModal() {
   }, [fetchSignedIframeUrl])
 
   return (
-    <Modal isOpen={fiatOnrampModalOpen} onDismiss={closeModal} maxHeight={720}>
+    <Modal isOpen={fiatOnrampModalOpen} onDismiss={closeModal} maxHeight={modalHeight} minHeight={modalHeight}>
```

Comment:
i'm basically copying the same approach that the currency search modal is using - calculate a `vh` value for the modal height which is mobile-compatible

Reply:
I think if this is the necessary solution you should prob add a `height` prop and pass it through the way max/min currently are:

![image](https://user-images.githubusercontent.com/5773490/234393203-d190a51e-e977-4d80-b7a7-bfb6a70f4bc6.png)

Reply:
sounds good!